extensions/ql-vscode/src/remote-queries/shared/analysis-result.ts

```diff
@@ -3,9 +3,28 @@ export type AnalysisResultStatus = 'InProgress' | 'Completed' | 'Failed';
 export interface AnalysisResults {
   nwo: string;
   status: AnalysisResultStatus;
-  results: QueryResult[];
+  results: AnalysisAlert[];
 }
 
-export interface QueryResult {
-  message?: string;
+export interface AnalysisAlert {
+  message: string;
+  severity: ResultSeverity;
+  filePath: string;
+  codeSnippet: CodeSnippet
+  highlightedRegion: HighligtedRegion
 }
+
+export interface CodeSnippet {
+  startLine: number;
+  endLine: number;
+  text: string;
+}
+
+export interface HighligtedRegion {
```

Comment:
🤏🏽 tiny typo 😄 (here and above)

```suggestion
export interface HighlightedRegion {
```

Reply:
🤦‍♀️